.travis.yml

```diff
@@ -36,5 +36,7 @@ script:
   - ./bin/console lint:yaml @CodeExplorerBundle
   # this checks that the Twig template files contain no syntax errors
   - ./bin/console lint:twig app/Resources @CodeExplorerBundle
+  # this checks that the XLIFF translations contain no syntax errors
+  - ./bin/console lint:xliff app/Resources
```

Comment:
What about more specific `app/Resources/translations`? We have translations only in that folder